grid/example/src/cells/RowEditOverlay.js

```diff
@@ -0,0 +1,215 @@
+import React, { memo, useState } from "react";
+
+const divStyle = {
+    backgroundColor: "#ccc",
+    height: "90px",
+    width: "1365px",
+    marginLeft: "-1370px",
+    marginTop: "-5px"
+}
+
+const segmetEditStyle = {
+    marginLeft: "240px",
+    marginTop: "-60px"
+}
+
+const weightPercentageStyle = {
+    width: "25%",
+    marginLeft: "410px",
+    marginTop: "-20px",
+}
+
+const weightValueStyle = {
+    width: "25%",
+    marginLeft: "410px",
+    marginTop: "6px",
+}
+const srStyle = {
+    marginLeft: "410px",
+    marginTop: "4px"
+}
+
+const editRemarksStyle = {
+    marginTop: "-75px",
+    marginLeft: "700px"
+}
+
+const buttonDivStyle = {
+    width: "15%",
+    marginTop: "-80px",
+    marginLeft: "1244px"
+}
+const RowEditOverLay = memo((props) => {
+    const { row, airportCodeList, updateCellData } = props;
+    const [value, setValue] = useState(row.original.flight);
+    const [cellSegmentValue, setCellSegmentValue] = useState(row.original.segment);
+    const [cellWeightValue, setCellWeightValue] = useState(row.original.weight);
+    const [srValue, setSrValue] = useState(row.original.sr);
+    const [remarksValue, setRemarksValue] = useState(row.original.remarks);
+
+
+    const onChangeSaveRemarks=(e)=>{
+    setRemarksValue(e.target.value)
+    }
+    const onChangeSaveSr = (e) => {
+        setSrValue(e.target.value);
+    };
+
+
+    const onWeightPercentageChange = (e) => {
+        setCellWeightValue({
+            ...cellWeightValue,
+            percentage: e.target.value
+        });
+    };
+
+    const onWeightValueChange = (e) => {
+        setCellWeightValue({
+            ...cellWeightValue,
+            value: e.target.value
+        });
+    };
+
+    const onChangeFrom = (e) => {
+        setCellSegmentValue({
+            ...cellSegmentValue,
+            from: e.target.value
+        });
+    };
+
+    const onChangeTo = (e) => {
+        setCellSegmentValue({
+            ...cellSegmentValue,
+            to: e.target.value
+        });
+    };
+
+    const onChangeSaveFlightNo = (e) => {
+        setValue({
+            ...value,
+            flightno: e.target.value
+        });
+    }
+
+    const onDateChange = (e) => {
+        setValue({
+            ...value,
+            date: getValueOfDate(e.target.value)
+        });
+    };
+
+    /**
+     * Method to Convert Date to required Format as per value of type
+     * @param {String} inputDate 
+     * @param {String} type 
+     */
+    const getValueOfDate = (dateValue, type) => {
```

Comment:
We can create a utility.js file in the example application, and lets place the getValueOfDate function there, and we will reuse them in all places (here and segment edit file).
Refer getData.js file in example application to see how to create, export and use re-usable functions

Reply:
moved to a utils package in DateUtility.js file